pom.xml

```diff
@@ -8,12 +8,35 @@
     <artifactId>Dealership</artifactId>
     <version>1.0-SNAPSHOT</version>
 
-    <dependencies><!-- https://mvnrepository.com/artifact/com.mysql/mysql-connector-j -->
+    <dependencies>
+
+        <!-- https://mvnrepository.com/artifact/com.mysql/mysql-connector-j -->
         <dependency>
             <groupId>com.mysql</groupId>
             <artifactId>mysql-connector-j</artifactId>
             <version>8.0.32</version>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/jakarta.xml.bind/jakarta.xml.bind-api -->
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>4.0.0</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.glassfish.jaxb/jaxb-runtime -->
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>4.0.3</version>
+        </dependency>
+
```

Comment:
I had some issues during marshaling because my date adapter was not being used. The date was not included in the xml files generated. While trying to resolve that I tried the jakarta import which contains the annotations for the model class, the marshaller, the unmarshaller and the JAXBContext. Combined with the the jaxb-runtime dependancy it provides a later version of JAXB. That was not the solution to my problem but i decided to keep those as it does not change the functionality of the in any way.

Reply:
Yes of course